.github/workflows/ci.yml

```diff
@@ -1,8 +1,35 @@
-on: pull_request
-
 name: CI
+on:
+
+  push:
```

Comment:
Currently it's not allowed to push master.

Reply:
done